### DISCUSSION TITLE
Call a webhook when certain queries change

### DISCUSSION THREAD

**Discussion by mmmcorpsvit** [meilisearch/product#736] [Feedback & Feature Proposal]
Hi, guys can help me please, i have users waiting to some goods (by conditions). How to subscribe user when new good add and this good follow user stored filter. Any idea? Meilisearch are any plans to implement subscribe functionality? Or maybe this future possible implement by side software? (sorry my English)

**Comment by macraig** [comment]
Hello @mmmcorpsvit! We don't have plans to implement a subscribe functionality for now, have you considered implementing this on your side of the application?

**Reply by mmmcorpsvit** [reply]
hm.... can share your vision of possible implementation?
or can show correct direction, task not very easy (from my first look)

**Reply by macraig** [reply]
Here's a potential proto solution you could try: 
- UI: Let users choose to be notified for a search request when there are new results, similar to how some housing and e-commerce websites do.
- Backend: Register the search request and link it to a user identifier (id / email). A cron job runs every day / every hour (depending on your user needs) and runs all of the registered search queries. Check if there are new search results for any of them. If there are new results, the system alerts the user via email, sms, etc.

We also have a very active [Discord community](https://discord.gg/5zzewNJr) you can check out!

**Reply by mmmcorpsvit** [reply]
ok, i was on Discord) and - "go to github and create topic)"

hm... in this case I need to store MeiliSearch results (or just CRC of topic and query) and compare to new results every time (by cron) hm..., yes, this can be the solution.)
